conference/cfp/templates/cfp/workshopsubmission_form.html

```diff
@@ -0,0 +1,22 @@
+{% extends CMS_TEMPLATE %}
+{% load cms_tags i18n %}
+
+{% block content %}
+    <section class="section-film-strip bg-purple">
+        <div class="grid">
+            <div class="s-10 m-10 l-8 centered">
+                {% static_placeholder "Content CFP" %}
```

Comment:
I'll call this "Content Workshop" instead of "Content CFP" to have a different text (``static_placehoder`` it's a django CMS thing to add content to a template)